_episodes/01-introduction.md

```diff
@@ -3,12 +3,202 @@ title: "Introduction"
+
+### What sort of problems can't it solve?
+
+* Only small amounts of training data
+* Tasks requiring an explanation of how the answer was arrived at
+* Being asked to classify things which are nothing like their training data.
+
+### What sort of problems can it solve, but shouldn't be used for?
+Deep learning needs a lot of computational power, for this reason it often relies on specialist hardware like graphical processing units (GPUs). Many computational problems can be solved using less intensive techniques, but could still technically be solved with deep learning.
+
+The following could technically be achieved using deep learning, but it would be a wasteful way to do it:
+
+* Logic operations, such as computing totals, averages, ranges etc.
+* Modelling well defined systems, where the equations governing them are known and understood.
+* Basic computer vision tasks such as edge detection, decreasing colour depth or blurring an image.
+
+
+> ## Application areas/History
+>
+> some benchmarks
+> DL is not new as a concept but availability of comp resources is now at stage that it can be done efficiently
+> some examples of interesting things that can be achieved using DL models:
+> image classification, text generation, language translation, GAN, Gameplay (Dota/Go), voice recognition,
+> self-driving cars, natural language and image processing, predictive forecasting, fraud detection in financial applications,
+> financial time-series forecasting, predictive and prescriptive analytics, medical image processing, power systems research,
+> recommendation systems
+> we are still far from computers understanding us
+>
+{: . callout}
+
+> ## What is Deep Learning:
+>
+> Give multiple definitions of Deep Learning and discuss;
+> OR
+> show both bubble representations of AI-ML-NN-DL and AI-ML-DL, ask which is right
+>
+{: .discussion}
+
+
+> ## Deep Learning Problems Exercise
+> Which of the following could you apply deep learning to?
+> 1. Recognising whether or not a picture contains a bird.
+> 2. Calculating the median and interquartile range of a dataset.
+> 3. Identifying pictures of people when only one or two images of them are available.
+> 4. Identifying people in pictures after being trained only on cats and dogs.
+> 5. Translating English into French.
+>> ## Solution
+>> 1 and 5 are the sort of tasks often solved with deep learning. 2. technically possible but deep learning is massively overkill, you could do the same with much less computing power using traditional techniques. 3. will fail because there's not enough training data. 4. will fail because the deep learning system only knows what cats and dogs look like, it might accidentally classify the people as cats or dogs.
+> {: .solution}
+{: .challenge}
+
+> ## What is DL - solution
+>
+> learning outcome -> it is vague, find your own
+> OR
+> hah! both are right in some ways, explain why NN has its place there as well
+>
+{: .solution}
+
+*the neuron/perceptron/unit/
```

Comment:
I don't think explanation on what neurons / perceptrons / units are should go in this episode. We didn't define it as one of the learning goals, and I believe that the intention of episode 2 is to explain what fully connected layers are, while explaining the concept of a neuron on the go.